spec/javascripts/fixtures/users.yml

```diff
@@ -0,0 +1,29 @@
+# Read about fixtures at http://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
+
+andrew:
+  github_id: 1060
+  access_token: <%= SecureRandom.hex(20) %>
+  github_login: 'andrew'
+
+tokenuser:
```

Comment:
Feels like we'd get a lot of mileage switching to factory girl now with all these very similar fixtures